docs/api/environment-variables.md

```diff
@@ -80,6 +80,16 @@ Don't attach to the current console session.
 
 Don't use the global menu bar on Linux.
 
+### `ELECTRON_DEFAULT_TRASH` _Linux_
```

Comment:
```suggestion
### `ELECTRON_TRASH` _Linux_
```

The environment variable is `ELECTRON_TRASH`, the C++ compiler macro used to define the default trash implementation is `ELECTRON_DEFAULT_TRASH`.

Otherwise looks good to me, thanks for working on this. :)

Reply:
oh whoops, thanks for catching this!